src/main/scala/com/gu/contentapi/Config.scala

```diff
@@ -4,6 +4,5 @@ object Config {
 
   val AudioLogsBucketName = "fastly-logs-audio"
 
-  val capiKey = sys.env("CAPI_KEY")
-
+  val capiKey = sys.env.getOrElse("CAPI_KEY", "INTERNAL-TIER-KEY-HERE")
```

Comment:
Just a way so the app doesn't blow up in case the env variable isn't set. It would make a lot more sense if I could use the `test` key as a default, but I need an internal tier key 😢